doc/source/guide/data_types/maps.rst

```diff
@@ -191,30 +193,30 @@ list::
     mc.maps[0]
 
 Mapcubes can hold maps that have different shapes.  To test if all the
-maps in a mapcube have the same shape::
+maps in a `~sunpy.map.MapCube` have the same shape::
 
     mc.all_maps_same_shape()
 
-It is often useful to return the image data in a mapcube as a single
-three dimensional numpy ndarray::
+It is often useful to return the image data in a `~sunpy.map.MapCube` as a single
+three dimensional Numpy `~numpy.ndarray`::
 
     mc.as_array()
 
 Note that an array is returned only if all the maps have the same
 shape.  If this is not true, an error (ValueError) is returned.  If all the
 maps have nx pixels in the x-direction, and ny pixels in the y-direction,
-and there are nt maps in the mapcube, the ndarray array that is
-returned has shape (ny, nx, nt).  The data of the first map in the mapcube 
-appears in the ndarray in position ``[:, :, 0]``, the data of second map in
-position ``[:, :, 1]``, and so on.  The order of maps in the mapcube is
-reproduced in the returned ndarray.
+and there are n maps in the mapcube, the `~numpy.ndarray` array that is
+returned has shape (ny, nx, nt).  The data of the first map in the `~sunpy.map.MapCube`
```

Comment:
"returned has shape (ny, nx, nt)" uses the variable "nt" which is not defined above.  Change "nt" to "n".